### ISSUE TITLE
Release Microsoft.CodeAnalysis.NetAnalyzers 7.0.1

### ISSUE THREAD

**Issue by rauhs** [dotnet/roslyn-analyzers#6502]
With the VS 17.5.0 release we again get a warning like:

```
packages\microsoft.codeanalysis.netanalyzers\7.0.0\build\Microsoft.CodeAnalysis.NetAnalyzers.targets(593,5): warning : The .NET SDK has newer analyzers with version '7.0.1' than what version '7.0.0' of 'Microsoft.CodeAnalysis.NetAnalyzers' package provide
```

can you release the 7.0.1 package?

We had the same annoying issue when we upgraded to 17.4. :/

**Comment by Cestbienmoi** [comment]
Any workaround (except remove the analyzer !) ? It is a lot of warnings for big solution !

**Comment by buckleyGI** [comment]
We have the same problem and don't tolerate too many warnings for too long.
Please coordinate these releases. We treat warnings as errors which we find best practice.

**Comment by mavasani** [comment]
You can disable this warning by defining this MSBuild property: https://github.com/dotnet/roslyn-analyzers/blob/c6352bf2e1bd214fce090829de1042000d021497/src/Tools/GenerateDocumentationAndConfigFiles/Program.cs#L1669

we’ll work on releasing the new NuGet package and also probably remove this warning

**Comment by buckleyGI** [comment]
Thanks for the swift response and the MSBuild property which at first sight is what we need @mavasani 
This message in our opinions shouldn't be a warning but more a notification.

**Comment by mavasani** [comment]
https://www.nuget.org/packages/Microsoft.CodeAnalysis.NetAnalyzers/7.0.1 is published now.
